.github/workflows/linux.yml

```diff
@@ -65,7 +66,7 @@ jobs:
       - name: Install git
         run: |
           apt-get update
-          apt-get install --assume-yes --no-install-recommends git ca-certificates
+          apt-get install --assume-yes --no-install-recommends git curl ca-certificates
```

Comment:
Is curl still required? looks it was needed to build sccache from sources only

Reply:
I removed it